### PR TITLE
test

### PR DESCRIPTION
**Pull request by CordlessWool**
# Description

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5703552&bid=18668238)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5703552&bid=18668238)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5703552&bid=18668238)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5703552&bid=18668238)

**Comment by janrenz** [comment]
@CordlessWool  Can this be closed?

**Comment by tofixx** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 27
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045527&bid=19807410)

File: test/mocha/content/content.test.js [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
 
 const assert = require('assert');
 const app = require('../../../app');
 const chai = require('chai');
-const expect = chai.expect;
+
+const { expect } = chai;
 const chaiHttp = require('chai-http');
+const { logger } = require('handlebars');
+const { logError } = require('gulp-sass');
```
Comment (tofixx) [inline test/mocha/content/content.test.js:9]: Codacy found an issue: ['logError' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045527)

File: test/mocha/content/content.test.js [hunk 2]
```diff
@@ -1,51 +1,56 @@
...
-            this.agent
-                .get('/content/')
-                .end((err, res) => {
-                    expect(res.statusCode).to.equal(200);
-                    expect(res.text).to.contain('Lern-Store');
-                    resolve();
-                });
-        });
-    });
+	it.only('GET /content', function () {
```
Comment (tofixx) [inline test/mocha/content/content.test.js:29]: Codacy found an issue: [Unexpected unnamed function.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045527)

**Pull request closed without merging**